lib/verbose.js

```diff
@@ -0,0 +1,19 @@
+import {debuglog} from 'node:util';
+import process from 'node:process';
+
+export const verboseDefault = debuglog('execa').enabled;
+
+const padField = (field, padding) => String(field).padStart(padding, '0');
+
+const getTimestamp = () => {
+	const date = new Date();
+	return `${padField(date.getHours(), 2)}:${padField(date.getMinutes(), 2)}:${padField(date.getSeconds(), 2)}.${padField(date.getMilliseconds(), 3)}`;
+};
+
+export const logCommand = (escapedCommand, {verbose}) => {
+	if (!verbose) {
+		return;
+	}
+
+	process.stderr.write(`[${getTimestamp()}] ${escapedCommand}\n`);
```

Comment:
Should be colorized, e.g. in bold?

This would look better. However, this would come with the following downsides:
  - Adding a color dependency, like `chalk` or (probably better here) `yoctocolors`
  - Would require color support detection, which can be done with `stderr.hasColors()`. Users can then use the `FORCE_COLOR` environment variable to override the default behavior.

I am not sure whether that's a good idea or not. What do you think?

Reply:
I think we should skip colors for now.

Reply:
Sounds good :+1: